src/Glpi/Api/HL/Middleware/CookieAuthMiddleware.php

```diff
@@ -50,7 +51,7 @@ public function process(MiddlewareInput $input, callable $next): void
         }
         // User could be authenticated by a cookie
         // Need to use cookies for session and start it manually
-        ini_set('session.use_cookies', '1');
+        Toolbox::iniSet('session.use_cookies', '1');
```

Comment:
This one should not be caught, beacause the feature cannot work without session cookies. However, it may fail if the value is already enforced to a truthy value in the PHP configuration.

```suggestion
        $use_cookies = filter_var(ini_get('session.use_cookies'), FILTER_VALIDATE_BOOLEAN);
        if ($use_cookies !== true) {
            ini_set('session.use_cookies', '1');
        }
```